Framework/API/inc/MantidAPI/Citation.h

```diff
@@ -0,0 +1,64 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#ifndef MANTID_API_CITATION_CITATION_H_
+#define MANTID_API_CITATION_CITATION_H_
+
+#include "MantidAPI/DllConfig.h"
+
+#include <string>
+#include <vector>
+
+#include <nexus/NeXusFile.hpp>
+
+namespace Mantid {
+namespace API {
+
+/*
+ * To construct a citation class give it properly formatted bibtex and endnote
```

Comment:
Can these be moved to Doxygen comments on the relevant constructors?

Reply:
I will leave it as a normal comment as Doxygen keeps trying to interpret the endnote string as commands

Reply:
Just remove the examples and link to an online reference.